administration/src/cards/CSVCard.test.ts

```diff
@@ -36,8 +37,7 @@ describe('CSVCard', () => {
     card.setValue('Kartentyp', 'Goldkarte')
 
     expect(card.fullName).toBe('Thea Test')
-    // TODO #1006: Ensure correct UTC Date for CSV Import, also remove the hours here
-    expect(card.expirationDate).toEqual(new Date(2022, 3, 3, 5))
```

Comment:
I believe, there is still a bug in prod (before this PR)?
I think we should not `expect(card.expirationDate).toEqual(new Date(2022, 3, 3, 5)` when we previously do `card.setValue('Ablaufdatum', '03.04.2022')`.

Reply:
But the Set value temporär adds 5 hours. I added this to avoid failing tests

Reply:
Yep, but the month actually differs: `Date(2022, 3, 3, 5)` vs `03.04.2022`

Reply:
Ah, nvmd, I was confused again, that `new Date` uses a monthIndex vs an actual month.
So this is less suspicious.

Reply:
hm thats true, but it works on production. Anyway i would like to merge your pr and create a new release before they start creating the sozialpass cards